static/src/javascripts/projects/common/modules/commercial/creatives/expandable-video-v2.js

```diff
@@ -0,0 +1,100 @@
+define([
+    'bean',
+    'bonzo',
+    'fastdom',
+    'common/utils/_',
+    'common/utils/$',
+    'common/utils/detect',
+    'common/utils/mediator',
+    'common/utils/storage',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/creatives/expandable-video-v2.html'
+], function (
+    bean,
+    bonzo,
+    fastdom,
+    _,
+    $,
+	detect,
+    mediator,
+    storage,
+    template,
+    svgs,
+    ExpandableVideoTpl
+) {
+
+    /**
+     * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10028247
+     */
+    var ExpandableVideo = function ($adSlot, params) {
+        this.$adSlot      = $adSlot;
+        this.params       = params;
+        this.isClosed     = true;
+
+        if (detect.isBreakpoint({min: 'tablet'})) {
+            this.closedHeight = 250;
+            this.openedHeight = 500;
+        } else {
+            this.closedHeight = 150;
+            this.openedHeight = 300;
+        }
+
+    };
+
+    ExpandableVideo.prototype.create = function () {
+        var videoHeight = this.openedHeight,
+            showmoreArrow = {
```

Comment:
I have been asked to use this standard in the past, which I think it's used throughout the site, so it's not my call to change this.